impl.nix

```diff
@@ -37,6 +38,7 @@ pkgs.mkShell rec {
         libGLU libGL
         glib
       ];
+    LD_PRELOAD = "${pkgs.gperftools}/lib/libtcmalloc.so";
```

Comment:
This shouldn't be necessary because of the LD_LIBRARY_PATH below. Does it not work if you leave this out?

Reply:
It does not find TCMalloc without the LD_PRELOAD (line 41). It does, however, work without the added gperftools in the hardware_deps (line 16).